cvrf2csaf/section_handlers/vulnerability.py

```diff
@@ -91,6 +105,75 @@ def _handle_remediations(root_element):
 
         return remediations
 
+    def _parse_score_set(self, score_set_element, mapping, version, json_property, product_status):
+        """Parses ScoreSetV2 or ScoreSetV3 element."""
+
+        cvss_score = {}
+
+        # Parse all input elements except ProductID
+        # note: baseScore is always present since it's mandatory for the input
+        for cvrf, csaf in mapping.items():
+            if hasattr(score_set_element, cvrf):
+                cvss_score[csaf] = getattr(score_set_element, cvrf).text
+
+        cvss_score = {csaf: getattr(score_set_element, cvrf).text for cvrf, csaf in mapping.items() if hasattr(score_set_element, cvrf)}
```

Comment:
Aren't these two doing the same?

Reply:
🤦 yes

Reply:
Why do we do that twice?

Reply:
I forgot to remove the old code - will be fixed.